### DISCUSSION TITLE
Kurdish language was successfully added 100%

### DISCUSSION THREAD

**Discussion by iamyahia** [photopea/photopea#6159] [Q&A]
Hello, amazing photopea team:hand: I wish you a wonderful day.

**Kurdish** language ( **کوردی ناوەندی** ) was _completely added_ in [translate section](https://www.photopea.com/translate/), 
 but we **cannot** see it in the language section on main [website](https://www.photopea.com)! why ??
Are there any other **steps** to be taken?

<img src="https://github.com/photopea/photopea/assets/90482598/36338047-69d2-452b-8f04-1b59a58e6811" width=40% height=20% />
<img src="https://github.com/photopea/photopea/assets/90482598/9cbe9d59-0716-4138-9bd5-ae4da4626525" width=40% height=40% />

**Comment by photopea** [comment]
I just added it, thanks a lot for your work! I synchronize photopea.com/translate with Photopea.com about once a month :)

Also, the Tagalog translation was finished, so we have 46 languages now.
